tests/test_create_pod_attaching_multiple_pvc_start_IO.py

```diff
@@ -0,0 +1,389 @@
+)
+SC = ocs.ocp.OCP(
+    kind='StorageClass', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+PVC = ocs.ocp.OCP(
+    kind='PersistentVolumeClaim', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+POD = ocs.ocp.OCP(
+    kind='Pod', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+OCP = ocs.ocp.OCP(
+    kind='Service', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+DC = ocs.ocp.OCP(
+    kind='DeploymentConfig', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+TEMP_YAML = os.path.join("templates/ocs-deployment", "temp.yaml")
+TEMPLATES_DIR = "templates/CSI/rbd"
+CIRROS_DIR = "templates/app-pod-yamls"
+TIMEOUT = 180
+SLEEP_TIME = 20
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    pytest fixture function
+    """
+    self = request.node.cls
+
+    def finalizer():
+        teardown(self)
+    request.addfinalizer(finalizer)
+    setup()
+
+
+def setup():
+    """
+    Setting up the environment for the test
+    """
+    create_ceph_block_pool()
+    create_rbd_secret()
+    create_rbd_storage_class()
+
+
+def teardown(self):
+    """
+    Teardown the created Environment
+    """
+    log.info(f"Cirros pod name {self.cirros_name[0]}")
+    delete_pod(self.cirros_name[0])
+    delete_persistent_volume(self.pv_list)
+    delete_storage_class()
+    delete_pool()
+    delete_secret()
+    os.remove(TEMP_YAML)
+
+
+def create_ceph_block_pool():
+    """
+    Create ceph block pool
+    """
+    template = os.path.join(TEMPLATES_DIR, "CephBlockPool.yaml")
+    log.info(f"Create a ceph block pool")
+    data = templating.generate_yaml_from_jinja2_template_with_data(
+        template
+    )
+    ceph_block_pool = data['metadata']['name']
+    with open(TEMP_YAML, 'w') as yaml_file:
+        yaml.dump(data, yaml_file)
+    assert CBP.create(yaml_file=TEMP_YAML)
+    open(TEMP_YAML, 'w').close()
+    sleep(5)  # Added sleep to avoid timing issue of resource creation
+    # Check for Pool creation succeeded in OC
+    try:
+        CBP.get(ceph_block_pool)
+        log.info(f"Ceph Block Pool {ceph_block_pool} creation succeeded")
+    except exceptions.CommandFailed:
+        assert log.info(
+            f'Failed to create {ceph_block_pool}'
+        )
+        return False
+
+
+def create_rbd_secret():
+    """
+    Get rbd secret from ceph and create secret in oc
+    """
+    secret_data = {}
+    secret_data['base64_encoded_admin_password'] = \
+        ocs.ocp.getbase64_ceph_secret("client.admin")
+    template = os.path.join(TEMPLATES_DIR, "secret.yaml")
+    log.info(f"Create secret for RBD storageclass")
+    data = templating.generate_yaml_from_jinja2_template_with_data(
+        template, **secret_data
+    )
+    secret_name = data['metadata']['name']
+    with open(TEMP_YAML, 'w') as yaml_file:
+        yaml.dump(data, yaml_file)
+    assert SEC.create(yaml_file=TEMP_YAML)
+    open(TEMP_YAML, 'w').close()
+    sleep(5)  # Added sleep to avoid timing issue of resource creation
```

Comment:
@RazTamir @ebenahar as discussed in google chat regarding get() function failure due to delay, am ok to remove the get() part of the code, since in case of failure there is an assert, but if the create command succeeds and if service took some delay then the next service/resource creation will fail.
anyway, let me know, If we are good here to remove the get() and exception handling.